examples/AvroGeneric/Program.cs

```diff
@@ -79,23 +79,28 @@ static void Main(string[] args)
             using (var consumer = new Consumer<string, GenericRecord>(consumerConfig, new AvroDeserializer<string>(), new AvroDeserializer<GenericRecord>()))
             using (var producer = new Producer<string, GenericRecord>(producerConfig, new AvroSerializer<string>(), new AvroSerializer<GenericRecord>()))
             {
-                consumer.OnMessage += (o, e)
-                    => Console.WriteLine($"Key: {e.Key}\nValue: {e.Value}");
-
                 consumer.OnError += (_, e)
                     => Console.WriteLine("Error: " + e.Reason);
 
-                consumer.OnConsumeError += (_, e)
-                    => Console.WriteLine("Consume error: " + e.Error.Reason);
-
                 consumer.Subscribe(topicName);
 
                 CancellationTokenSource cts = new CancellationTokenSource();
```

Comment:
I am not the guy who can tell what the styleguide will be but would be good if everything in examples is written in more .NET way for instance using `var` and omitting redundant type names.